crates/solidity/inputs/language/bindings/rules.msgb

```diff
@@ -2543,6 +2545,25 @@ inherit .star_extension
   edge @type_expr.output -> typeof
   edge typeof -> type
   edge type -> @type_expr.lexical_scope
+
```

Comment:
does it make sense to try to abstract this with the typeof(enum) case?

Reply:
Not really. Both apply in different contexts (ie. the enum definition vs. the `type()` expression), but more importantly, the version where `min/max` are valid are different.